lib/scalingo/regional/addons.rb

```diff
@@ -80,6 +80,21 @@ def sso(app_id, addon_id, headers = nil, &block)
       unpack(:addon) { response }
     end
 
+    def authenticate(app_id, addon_id, headers = nil, &block)
```

Comment:
I would change either the name (or add a `!` at the end), to highlight that this method is more than a request/response (since 99% of methods in this context)

Reply:
Will do.

I also still have some spec to add before merging. Will ping you once it is done for a final short review

Reply:
@ksol I've done.
Is it still all good?

Reply:
Yep, looks fine!